juju/juju
# 2.5 - Implement InstanceFirewaller for OCI Instances

## Description of change

This patch implements the `InstanceFirewaller` interface on the `ociInstance` type. It uses the `SshInstanceConfigurator` to manipulate IP tables via SSH.

## QA steps

The `assess-network-health` functional test will pass for the OCI provider. In manual steps this is:
- Bootstrap to OCI
- `juju deploy ubuntu -n 2`.
- `juju deploy '~juju-qa/network-health'`.
- `juju expose ubuntu`.
- `juju expose network-health`.
- `juju add-relation ubuntu network-health`.
- `juju run --unit ubuntu/0 curl <public IP of machine 1>:8039`.
- Ensure the result is "pass".

None.

## Bug reference

N/A
